jest.config.ts

```diff
@@ -0,0 +1,14 @@
+import type { Config } from 'jest';
+
+const config: Config = {
+  preset: 'ts-jest',
+  transform: {
+    '^.+\\.(ts|tsx)?$': ['ts-jest', { 
+      "tsconfig": "./packages/website/tsconfig.json"
```

Comment:
We don't want to use a specific package's `tsconfig` file, since this should be usable for any TypeScript project we may have.